sklearn/ensemble/iforest.py

```diff
@@ -309,7 +286,7 @@ def fit(self, X, y=None, sample_weight=None):
         # else, define offset_ wrt contamination parameter, so that the
```

Comment:
the part after the comma can be removed in the comment as `threshold_` is not used anymore